examples/cute/tutorial/sgemm_1_sycl.cpp

```diff
@@ -376,6 +376,14 @@ int main(int argc, char** argv) {
   char transB = 'T';
   if (argc >= 6) sscanf(argv[5], "%c", &transB);
 
+  sycl::property_list prop = {
+          sycl::property::queue::in_order(),
+          sycl::property::queue::enable_profiling()
+  };
+
+  auto q = sycl::queue(syclcompat::get_default_context(), syclcompat::get_current_device(), prop);
```

Comment:
There is a constructor overload for the queue class which accepts the property list and uses the default device selector, so you will not need to pass context and device along with the property 

> explicit queue(const property_list& propList = {})

Reply:
Also, we can just refactor the queue creation in a separate functions, which accepts a property list, and returns the queue. That can later on be moved to syclcompat mostly as is

Reply:
SYCLCompat already supports this but it does not work because of a bug.

```c++
#ifdef SYCLCOMPAT_PROFILING_ENABLED
    prop.push_back(sycl::property::queue::enable_profiling());
#endif
```